aws/logs_monitoring/parsing.py

```diff
@@ -241,10 +241,14 @@ def parse_event_source(event, key):
 def find_cloudwatch_source(log_group):
     # e.g. /aws/rds/instance/my-mariadb/error
     if log_group.startswith("/aws/rds"):
-        for engine in ["mariadb", "mysql"]:
-            if engine in log_group:
-                return engine
-        return "rds"
+        return next(
```

Comment:
Could you revert this back to the older format using the `if` statement? I think it's a bit more obvious that way.

Reply:
sure, no problem. I changed it back